src/QirRuntime/lib/QIR/callables.cpp

```diff
@@ -138,7 +139,7 @@ struct QirCallable
         {
             if (this->controlledDepth > 0)
             {
-                throw new std::exception("multi-level controlled operations not implemented yet");
+                throw new std::runtime_error("multi-level controlled operations not implemented yet");

```

Comment:
This is the build-break on Linux that caused the discovery of the fact that CI build isn't running :)